### PR TITLE
Remove torchao_nightly package

### PR DESCRIPTION
**Pull request by atalman**
Final nightly package name similar to this: ``torchao-0.4.0.dev20240812+cpu-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl``

### CODE REVIEW

**Review by msaroufim — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 from setuptools import find_packages, setup
 
-current_date = datetime.now().strftime("%Y.%m.%d")
+current_date = datetime.now().strftime("%Y%m%d")
```
Comment (msaroufim) [inline setup.py:13]: heads up to @joecummings and @ebsmothers in case you have version guards on nightly date

Comment (joecummings) [reply setup.py:13]: Yep, we'll catch and fix

**Comment by msaroufim** [comment]
Feel free to merge whenever you're ready @atalman - presumably the right test plan is to trigger this once and see all the published nightly binaries for today on the pytorch index?

**Comment by joecummings** [comment]
When this does land @atalman,  can you ping me so I can fix our tests real quick?

**Comment by jerryzh168** [comment]
does `pip install torchao-nightly` work after this PR?

**Pull request merged**